webui/react/src/pages/Cluster/ClusterOverview.tsx

```diff
@@ -107,8 +112,10 @@ const ClusterOverview: React.FC = () => {
   }, []);
 
   useEffect(() => {
+    fetchAgents();
```

Comment:
agent fetch is already being polled at the top level of Navigation, but it makes sense for it to get called at lease during a mount for cluster overview, otherwise the info can be pretty out of sync right off the bat.